src/plotman/archive.py

```diff
@@ -96,7 +96,7 @@ def spawn_archive_process(dir_cfg: configuration.Directories, arch_cfg: configur
 
 def compute_priority(phase: job.Phase, gb_free: float, n_plots: int) -> int:
     # All these values are designed around dst buffer dirs of about
-    # ~2TB size and containing k32 plots.  TODO: Generalize, and
+    # ~2TB size and containing k plots.  TODO: Generalize, and
```

Comment:
I suspect this is still a valid reference to k32.

```suggestion
    # ~2TB size and containing k32 plots.  TODO: Generalize, and
```